Makefile

```
@@ -32,8 +32,7 @@ DOCKER_BUILD          := $(DOCKER) build
 DOCKER_PUSH           := $(DOCKER) push
 DOCKER_COMPOSE        := docker-compose
 
-
-DOCKER_REPO           :=  meltwater/drone-cache
+CONTAINER_REPO        :=  meltwater/drone-cache
```

Also, I think to allow customization of `CONTAINER_REPO` is better
```suggestion
CONTAINER_REPO        ?=  meltwater/drone-cache
```

---

👍